modules/issue_tracker/php/issue.class.inc

```diff
@@ -27,7 +25,7 @@ require_once __DIR__ . '/issue_tracker_ControlPanel.class.inc';
  * @link     https://github.com/aces/Loris-Trunk
  */
 
-class NDB_Form_Issue_Tracker extends NDB_Form
+class Issue extends \NDB_Form
```

Comment:
I can't comment because it's the line below here, but I'm surprised the overriding display to return null works. For the media module I had to remove that hack and set the skipTemplate class variable to true in order to get format=json to work..

Reply:
I would guess that it is due to extending NDB_Form instead of NDB_Menu_Filter